src/Broadway/Domain/DomainMessageHasMailsDisabled.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Broadway\Domain;
+
+use Broadway\Domain\DomainMessage;
+
+class DomainMessageHasMailsDisabled implements DomainMessageSpecificationInterface
+{
+    public const METADATA_MAILS_DISABLED_KEY = 'disable_mails';
+
+    public function isSatisfiedBy(DomainMessage $domainMessage): bool
+    {
+        $metadata = $domainMessage->getMetadata()->serialize();
```

Comment:
Nit: I wonder if we could just use the `get` method on the `Metadata` class here?